### PR TITLE
Feature/update grid

### PR DESCRIPTION
**Pull request by bbakernoaa**
This just updates the `config/cmaq/HEMCO_sa_Grid.rc` file to include a larger domain.  It also corrects the OMI-HTAP filename in the config file.

### CODE REVIEW

File: config/cmaq/HEMCO_sa_Grid.rc [hunk 1]
```diff
@@ -1,8 +1,8 @@
...
-XMIN: -140.0
-XMAX: -50.0
-YMIN:  20.0 
-YMAX:  65.0 
-NX: 900
-NY: 450
+XMIN: -160.0
+XMAX:  -30.0
+YMIN:  7
+YMAX:  82
```
Comment (zmoon) [inline config/cmaq/HEMCO_sa_Grid.rc:5]: Any particular reason no `.0`s on these?

File: config/cmaq/HEMCO_sa_Grid.rc [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+NX: 1250
```
Comment (zmoon) [inline config/cmaq/HEMCO_sa_Grid.rc:6]:
```suggestion
NX: 1300
```
if want to be consistent with what it was like before

**Comment by zmoon** [comment]
Also, are the MIN/MAX/N for edges or centers? Just curious.

Update: looks like the spacing is computed using (MAX - MIN) / N, so should add +1 to the N's if 0.1 degree is the goal?

**Comment by bbakernoaa** [comment]
These are already a little larger than the target domain. 👍

**Pull request merged**